src/routes/homepage-redesign-preview/index.svelte

```diff
@@ -1,106 +1,260 @@
+    }
+  }
+
+  .cta-card {
+    width: 350px;
+
+    @include media(">large") {
+      float: right;
+    }
+
+    @include media("<=medium") {
+      margin-top: 24px;
+    }
+  }
+
+  .icon-circle {
+    svg {
+      fill: red;
+      color: red;
+    }
+  }
+</style>
+
 <svelte:head>
   <title>Cal-Adapt</title>
 </svelte:head>
 
+<!-- Banner -->
 <Banner
-  titleText="Cal-Adapt provides public access to climate data that contribute to
-    California’s Climate Change Assessments."
-  subtitleText="Cal-Adapt provides interactive, easy-to-use visualization and
-    data download tools for a broad range of stakeholders; a “pro” version for
-    users with highly technical, data-intensive needs; and links to state-funded
-    grants that support development of data for Cal-Adapt."
+  titleText="Explore and analyze climate data from California’s Climate Change Assessments"
+  subtitleText="Cal-Adapt provides the public, researchers, government agencies and industry stakeholders with essential data & tools for climate adaptation planning, building resiliency, and fostering community engagement."
+  bannerImg="/img/banners/yosemite_1600x540.jpg"
+  bannerImgMobile="/img/banners/yosemite_700x800.jpg"
+  overlayOpacity="{0.4}"
   titleFontSize="2rem"
   titleFontWeight="600"
+  iconPaths="{icons}"
   useOffset="{false}"
-  bannerImg="linear-gradient(180deg, #003850 0%, #067C74 100%)"
-  overlayOpacity="0"
-/>
+>
+  <!--   <div class="btn-container" slot="button">
+    <Button icon="{ArrowRight16}" href="/about" sapper:prefetch
+      >More about Cal-Adapt</Button
+    >
+  </div> -->
+  <div class="cta-card" slot="cta-card">
+    <Card
+      {...{
+        titleText: "Cal-Adapt is evolving!",
+        linkPath: `/help/`,
+        description:
+          "Learn about the Cal-Adapt enterprise and our mission to support California's climate change initiatives and preview our future plans.",
+        height: 18,
+        ctaText: "Read More",
+      }}
+    >
+      <div class="icon-circle bg-teal-60" slot="icon_slot">
+        <svelte:component this="{InformationFilled32}" fill="red" />
+      </div>
+    </Card>
+  </div>
+</Banner>
 
 <div class="spacing--v-32"></div>
 
-<Grid>
-  <Row>
-    <Column lg="{{ span: 12 }}">
+<div class="bx--grid">
+  <div class="bx--row">
+    <div class="bx--col-lg-12 bx--col-md-8 bx--col-sm-4">
+      <!-- Fifth Assessment Notification -->
+      <div class="alert">
+        <InlineNotification
+          lowContrast
+          hideCloseButton
+          kind="info"
+          title="Looking for climate data for California's Fifth Climate Change Assessment?"
+        >
+          <p class="bx--inline-notification__subtitle">
+            Visit the <a href="/data">Cal-Adapt Analytics Engine</a>
+          </p>
+        </InlineNotification>
+      </div>
+
+      <!-- Fourth Assessment Block -->
+      <h2>Explore interactive maps and charts</h2>
+      <div class="spacing--v-24"></div>
+      <p class="full-width">
```

Comment:
I'm a little conflicted about making this `p` full width of the parent div now that I see it on the page instead of in FigmaI find paragraphs with shorter lengths easier to scan and read. The default `p` width is `75ch`. Maybe we can experiment with increasing the number of `ch` instead of setting it to `100%`.